lua/nvim-lightbulb.lua

```diff
@@ -24,6 +24,10 @@ local function _update_float(opts)
     if opts.win_opts["anchor"] ~= nil then
         vim.fn.nvim_win_set_config(win, { anchor = opts.win_opts.anchor })
     end
+
+    if opts.win_opts["winblend"] ~= nil then
```

Comment:
```suggestion
    if opts.win_opts["transparency"] ~= nil then
```

Reply:
I kinda disagree. There is already a name for this the user might be used to as a NeoVim user. Also reads the code confusing because we read that `winblend` gets set to `transparency`. But if you insist I will change it of course. Just wanted to raise my concerns.

Reply:
Good point, let's go with `winblend` then, and have the README change only.